api_docs/changelog.md

```diff
@@ -25,6 +25,12 @@ format used by the Zulip server that they are interacting with.
 * [`POST /users/me/subscriptions`](/api/subscribe): Users belonging to
   `can_add_subscribers_group` should be able to add subscribers to a
   private channel without being subscribed to it.
+* [`DELETE /users/me/subscriptions`](/api/get-subscriptions): Channel
+  administrators can now unsubscribe other users even if they are not
+  an organization administrator or part of
+  `can_remove_subscribers_group`. Realm administrators are now
+  allowed to unsubscribe other users from a private channel even if
+  they are not subscribed to that channel.
```

Comment:
Realm administrators already could; `allow_realm_admin=True` was passed to `check_basic_stream_access`. Code is correct, just the documentation is inaccurate.